dev/scf/README.md

```diff
@@ -4,18 +4,72 @@ The targets under `scf` are used to apply the rendered Helm template to a Kubern
 Any `*values.yaml` files under this directory are ignored by git, but used by Bazel to render the
 SCF chart before applying to the cluster with kubectl.
 
+## Prerequesites
+
+You need to have [cf-operator] installed in your Kubernetes cluster (currently
+tip of `master` is required).
+
+[cf-operator]: https://github.com/cloudfoundry-incubator/cf-operator
+
+### Building cf-operator
+
+Build the operator image from a `cf-operator` checkout:
+
+```sh
+SAVE_TARBALL=true make build-image
+```
+
+### Installing cf-operator
+
+Load the cf-operator, depending on if you're using `kind` or `minikube`:
+
+#### When using `kind`
+
+Load the cf-operator image in `kind`:
+
+```sh
+kind load image-archive --name scf binaries/cf-operator-image.tgz
+```
+
+#### When using `minikube`
+
+Load the cf-operator image in `minikube`:
+
+```sh
+eval $(minikube docker-env)
```

Comment:
I'd prefer giving this instruction in a subshell, so the `eval` doesn't persist:

```
(eval $(minikube docker-env); docker load --input binaries/cf-operator-image.tgz)
```

Reply:
Changed in 23e13e9231af4d88ea31f53bd0888e3fb2bcb86c.